### ISSUE TITLE
Next JS - Revalidate missing from some pages

### ISSUE THREAD

**Issue by multivoltage** [aws-amplify/amplify-hosting#3617]
### Before opening, please confirm:

- [X] I have checked to see if my question is addressed in the [FAQ](https://github.com/aws-amplify/amplify-hosting/blob/master/FAQ.md).
- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-hosting/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-hosting/blob/master/CONTRIBUTING.md).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I have removed any sensitive information from my code snippets and submission.

### App Id

d2skhgz4a0rx0t

### AWS Region

eu-west-1

### Amplify Hosting feature

Deployments, Frontend builds

### Frontend framework

Next.js

### Next.js version

12

### Next.js router

Pages Router

### Describe the bug

We migrated to next 12 with web-compute month ago. We had issue but since 2 month all work well.

Since today (25 july) in some page in my stack transation from server side are missing.
We noticed that after a deploy (of same build of last days).

We use `next-i18next` for transations. For pages with `revalidate` or pages using `getServerSideProps` options it seems that transations are missing.

Other pages witt getStaticProps and without revalide works. using next-i18-next require this code for pages:

```js
export const getStaticProps: GetStaticProps = async () => {
    const locale: T_Locale = "it";

    return {
        props: {
            locale,
            ...(await serverSideTranslations("it", ["common"])),
        },
    };
};
```

### Expected behavior

1- Translations are available in all pages.
2. I can see a log with error

### Reproduction steps

Init a next 12 with pages directory app
install next-i18-next.

Create a page with revalidate.

This happend only on amplify

### Log output

<details>

```
# Put your logs below this line


```

</details>

**Comment by ferdingler** [comment]
Hi @multivoltage, I'm not very familiar with the `next-i18-next` library. Where do you put your translation `json` files? Are they located in the `public` folder?

**Comment by victorccccc** [comment]
Hi, I cannot reproduce your issue by setting `revalidate` or using `getServerSideProps`. This is my test app: https://main.d1rhug8t3v6e4r.amplifyapp.com and

I'm using example app from `next-i18-next`: https://github.com/Victorccding/simple-i18n-next

Can you help me to reproduce the issue? Thanks

**Comment by multivoltage** [comment]
@victorccccc thanks.
I fixed setting localPath to path resolve(something), like the docs write.
But I'm very confused since amplify change something and this fix become mandatory in my codebase.

Can you provide some info about a change?

**Comment by shikixyx** [comment]
I was facing the same problem a few days ago, but setting up localePath in next-i18next.config.js solved the problem.

**Comment by multivoltage** [comment]
Yes, but did you solve in a "random way" reading stackoverflow or you read some log in some point?

**Comment by shikixyx** [comment]
Yes, I found it in a "random way".

**Comment by Jay2113** [comment]
Closing the thread since we were not able to reproduce this issue and based on the previous comments it seems to have been resolved.
